src/components/RadioInput/index.js

```diff
@@ -1,21 +1,68 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import { Form } from 'semantic-ui-react';
-import { TextError } from '../../styles/App';
+import { Emoji } from 'emoji-mart';
 
-const RadioInput = ({
-  id,
-  label,
-  children,
-  error,
-  touched
-}) => (
-  <Fragment>
-    <Form.Group inline>
-      <label htmlFor={id}>{label}</label>
-      {children}
-    </Form.Group>
-    {touched && <TextError>{error}</TextError>}
-  </Fragment>
-);
+class RadioInput extends React.Component {
+
+  handleRadioChange = (e, { value }) => {
+    const { onRadioChange } = this.props;
+    onRadioChange(value);
+  };
+
+  render() {
+    const { emotion } = this.props;
+    return (
+      <Form.Group inline>
+        {/*eslint-disable-next-line*/}
+        <label htmlFor="Emotion">Emotion</label>
+
+        <Form.Radio
+          label={(
+            // eslint-disable-next-line
```

Comment:
I think that if you feel some specific esLint rule feels noisy, it's better to change `.eslintrc` and avoid using // `// eslint-disable-next-line`.

Because we will probably need to input `disable-next-line` even in pages we will implement later

Reply:
I'll fix this in later PR